tests/test_add_rectangle.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2025 Tsolo.io
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from itertools import combinations
+from pathlib import Path
+
+import pytest
+
+from cycax.cycad import Print3D
+from cycax.cycad.engines.part_build123d import PartEngineBuild123d
+
+
+def test_cube_add_sphere(tmp_path: Path):
+    cube = Print3D(x_size=11, y_size=11, z_size=11, part_no="testing_add_rectangle")
+
+    edges = combinations(("LEFT", "RIGHT", "FRONT", "BACK", "TOP", "BOTTOM"), 2)
```

Comment:
Lest use the constants and not the strings: `edges = combinations((LEFT, RIGHT, FRONT, BACK, TOP, BOTTOM), 2)`